src/relay/backend/contrib/cmsisnn/relay_to_tir.cc

```diff
@@ -194,7 +180,7 @@ class RelayToTIRVisitor : public MixedModeMutator {
     if (depth_multiplier != -1) {
       cmsisnn_api = "arm_depthwise_conv_wrapper_s8";
       Array<PrimExpr> depthwise_filter_shape{1, filter_shape[0], filter_shape[1], out_channels};
```

Comment:
Is it possible to use something like 
`
int kernel_pos_o = kernel_layout.find("O");
`
instead of the numbers ?